### ISSUE TITLE
Ten onrechte melding `Identifiable construct must assign yes to the tagged value "Indicatie kerngegeven"`

### ISSUE THREAD

**Issue by melsk-r** [Imvertor/Imvertor-Maven#486]
Bij het verwerken van een van mijn UGM's wordt de melding:

`Identifiable construct must assign yes to the tagged value "Indicatie kerngegeven"`

gegeven. De tagged value op de betreffende attributen heeft echter gewoon, zoals in het metamodel gedefinieerd, de waarde 'Ja' en deze melding zou dan ook niet voor moeten komen.

**Comment by ArjanLoeffen** [comment]
Ik zie dat deze melding voorkomt bij attribuutsoorten met
- de tagged value CFG-TV-INDICATIEMATCHGEGEVEN, waarbij de waarde Ja/Yes is, én
- het attribuut de imvert:is-id staat op Yes.

Ik zie dat INDICATIEMATCHGEGEVEN de nederlandse vorm "Indicatie matchgegeven" heeft (met Ja of Nee als mogelijke waarde, zie /KING/cfg/tvsets/KINGUGM.xml), dus dat is verwarrend. Weet je zeker dat dit de melding is, en zo ja, kun  je me het eens laten zien?

**Comment by melsk-r** [comment]
Ik ben vergeten hier het jobnummer bij te vermelden dus ik zal deze week even kijken of ik hem kan reproduceren en je dan het jobnummer in dit issue vermelden.

**Comment by melsk-r** [comment]
@ArjanLoeffen Zie [deze job](https://imvertor.armatiek.nl/imvertor-executor/dashboard/app?subpath=UGM%2FTEST%20UGM%20VOERTUIGEN%2F20240523).

**Comment by ArjanLoeffen** [comment]
Het betreft deze melding:
> Identifiable construct must assign yes to the tagged value "Indicatie matchgegeven" 
En dat klopt. 
Voor betreffende attribuutsoort:
> TEST UGM VOERTUIGEN Model Entiteittype::parkeervergunning.parkeervergunningnummer (attrib)
is aangegeven dat het de identifier betreft (imvert:is-id eq true). Dat is bepaald op basis van de isID property in het EA  model.

Dus de melding is terecht.

**Comment by melsk-r** [comment]
Nee, de melding is niet terecht. De melding zou alleen terecht zijn als de tagged value 'Indicatie matchgegeven' op de betreffende attributen de waarde 'Nee' of helemaal geen waarde zou hebben (de default waarde is immers 'Nee'). Dat is hier echter niet het geval, de tagged value heeft immers op alle 4 de betreffende attributen de waarde 'Ja'.

**Comment by ArjanLoeffen** [comment]
Aha, Gevonden. Check:

https://github.com/Imvertor/Imvertor-Maven/blob/aca580aee355793b89e7b8afbcbb672f3bfb8682/src/main/resources/xsl/Validator/Imvert2canonical-KINGUGM.xsl#L81

Hier wordt feitelijk de waarde van matchgegeven overgeslagen wanneer er een indicatie kerngegeven is. Dit is een tijdelijke situatie wellicht: zie de redmine referentie. Als je indicatie kerngegeven leeglaat wordt matchgegeven uitgelezen. Of je heft deze fix op, dus verwijderen uit canonisering voor UGM's..

**Comment by melsk-r** [comment]
We moeten hier sowieso een keer naar kijken want volgens mij is dit een erfenis uit het StUF verleden. Zowel Kerngegevens als Matchgegevens zijn voor OAS specificaties denk ik niet van toepassing.

**Comment by ArjanLoeffen** [comment]
Dus niks doen?

**Comment by melsk-r** [comment]
Ik ga even met Johan overleggen.
